### PR TITLE
Expose Ground Truth SITL position

### PR DESCRIPTION
**Pull request by kamilritz**

### CODE REVIEW

File: src/plugins/telemetry/include/plugins/telemetry/telemetry.h [hunk 1]
```diff
@@ -122,6 +122,17 @@ class Telemetry : public PluginBase {
         float yaw_rad_s; /**< @brief Yaw angular velocity */
     };
 
+    /**
+     * @brief Ground truth type.
+     *
+     * Ground truth position information available in simulation
```
Comment (julianoes) [inline src/plugins/telemetry/include/plugins/telemetry/telemetry.h]:
```suggestion
     * Ground truth position information available in simulation.
```

File: src/integration_tests/telemetry_async.cpp [hunk 2]
```diff
@@ -244,6 +252,14 @@ void print_fixedwing_metrics(Telemetry::FixedwingMetrics fixedwing_metrics)
     _received_fixedwing_metrics = true;
 }
 
+void print_ground_truth(Telemetry::GroundTruth ground_truth)
+{
+    std::cout << "Latitude: " << ground_truth.latitude_deg << " deg, "
+              << "Longitude: " << ground_truth.longitude_deg << " deg, "
+              << "Altitude: " << ground_truth.absolute_altitude_m << " m" << std::endl;
```
Comment (julianoes) [inline src/integration_tests/telemetry_async.cpp]: You could actually just do `std::cout << ground_truth << std::endl;` since you implemented it.

File: src/integration_tests/telemetry_sync.cpp [hunk 3]
```diff
@@ -26,7 +26,7 @@ TEST_F(SitlTest, TelemetrySync)
     EXPECT_EQ(telemetry->set_rate_gps_info(10.0), Telemetry::Result::SUCCESS);
     EXPECT_EQ(telemetry->set_rate_battery(10.0), Telemetry::Result::SUCCESS);
     EXPECT_EQ(telemetry->set_rate_actuator_control_target(10.0), Telemetry::Result::SUCCESS);
-    EXPECT_EQ(telemetry->set_rate_fixedwing_metrics(10.0), Telemetry::Result::SUCCESS);
+    EXPECT_EQ(telemetry->set_rate_ground_truth(10.0), Telemetry::Result::SUCCESS);
```
Comment (julianoes) [inline src/integration_tests/telemetry_sync.cpp]: Have you checked if this works if you don't enable the message in the startup script of PX4?

Comment (kamilritz) [reply src/integration_tests/telemetry_sync.cpp]: Yes.

File: src/integration_tests/telemetry_sync.cpp [hunk 4]
```diff
@@ -26,7 +26,7 @@ TEST_F(SitlTest, TelemetrySync)
     EXPECT_EQ(telemetry->set_rate_gps_info(10.0), Telemetry::Result::SUCCESS);
     EXPECT_EQ(telemetry->set_rate_battery(10.0), Telemetry::Result::SUCCESS);
     EXPECT_EQ(telemetry->set_rate_actuator_control_target(10.0), Telemetry::Result::SUCCESS);
-    EXPECT_EQ(telemetry->set_rate_fixedwing_metrics(10.0), Telemetry::Result::SUCCESS);
```
Comment (JonasVautherin) [inline src/integration_tests/telemetry_sync.cpp]: Why was this removed? :thinking:

Comment (kamilritz) [reply src/integration_tests/telemetry_sync.cpp]: 🤔 there is no good reason, sry for that. Will add it back.

Comment (julianoes) [reply src/integration_tests/telemetry_sync.cpp]: Oops, missed it.
